todo_app/data/trello_items.py

```diff
@@ -0,0 +1,49 @@
+import requests
+import os
+from todo_app.data.item import Item
+from todo_app.data.status import ItemStatus
+
+API_KEY = os.getenv('API_KEY')
+TOKEN = os.getenv('TOKEN')
+BOARD_ID = os.getenv('BOARD_ID')
+BASE_URL = 'https://api.trello.com/1'
+
+TODO_LIST_NAME = 'To Do'
+DOING_LIST_NAME = 'Doing'
+DONE_LIST_NAME = 'Done'
+
+def get_items():
+    lists = requests.get(f'{BASE_URL}/boards/{BOARD_ID}/lists', params={ 'cards': 'open', 'key': API_KEY, 'token': TOKEN}).json()
```

Comment:
Very minor point but since every request to Trello requires the key and token you could try commonising that part of `params`.

E.g. with a shared `credentials` dictionary you could write something like `params = { 'cards': 'open', **credentials }`